### DISCUSSION TITLE
Maybe a request: make the table cell Rich Text

### DISCUSSION THREAD

**Discussion by davidlehub** [directus/directus#16790] [Feature Requests]
Hide, I'm new to Directus and I think I'm gonna love it!

I wonder if it's possible to have rich text on table cell. I know, with WYSIWYG, we can set the color (foreground and background), Bold, etc, which is nice 👍 But I'm looking for a way to have each character different color and/or style for Text's cell.

Btw, sorry for my so so English :)
David

**Comment by paescuj** [comment]
Are you talking about tables inside the WYSIWYG field or the content table in Directus itself? The former is already possible:

<img width="758" alt="Screenshot 2022-12-12 at 17 59 04" src="https://user-images.githubusercontent.com/5363448/207106798-66a3d5f8-3588-4e98-a93e-48372a7e6780.png">

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
